### DISCUSSION TITLE
No libraries listed

### DISCUSSION THREAD

**Discussion by fliberdygibits** [seerr-team/seerr#341] [Q&A]
Running Jellyseerr 1.3.0 in docker.  While working on something else I happened to notice that in Settings > Jellyfin there are no libraries listed and none appear when I click the sync button.  The log is simply reporting "_Something went wrong while getting libraries from the Jellyfin server: Request failed with status code 401_".  I can't quite tell what effect this is having on jellyseerr operations and I also can't quite tell what I need to fix to get this working again.  Wonder if someone could point me in a direction?

Also just updated to 1.4.1 but that didn't do anything thus far.

**Comment by fallenbagel** [comment]
Let's try refreshing the access token. Could you log out and log back in again and then press sync?

**Reply by fliberdygibits** [reply]
Did that earlier, also re-deployed the container.  Did it again just now.  Nothing.

**Reply by fallenbagel** [reply]
Status code 401 means unauthorised tho 🤔
Could you try making a new admin account and try to setup (remember to point to a different config folder)
Also make sure you don't have any device limits set

**Comment by fliberdygibits** [comment]
What bit is the unauthorized part?  I was able to log in just fine.
Jellyseerr doesn't have an API key from jellyfin or anything.

On Mon, Mar 6, 2023 at 12:42 AM Fallenbagel ***@***.***>
wrote:

> Status code 401 means unauthorised tho 🤔
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Fallenbagel/jellyseerr/discussions/341#discussioncomment-5213028>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOWBYVUBZSAQKMIC776M25LW2WPQXANCNFSM6AAAAAAVQTWMZ4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
I'll be more willing to encourage thinking outside the box when I see some
evidence of thinking inside the box.
-Terry Pratchett

**Reply by fallenbagel** [reply]
It seems to be unable to retrieve the libraries using the admin account you set it up with. And accessing that endpoint is throwing an unauthorised 401

**Comment by fliberdygibits** [comment]
I added ldap to my jellyfin earlier today and changed the password but I
changed it to the same password in jellyseerr as well.... tho to be fair
I'm not sure if that's the admin account you're referring to?  My primary
account listed as "owner" in jellyseerr?  If that's not it, what other
account did I setup for that?

On Mon, Mar 6, 2023 at 1:09 AM Fallenbagel ***@***.***> wrote:

> It seems to be unable to retrieve the libraries using the admin account
> you set it up with. And accessing that endpoint is throwing an unauthorised
> 401
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Fallenbagel/jellyseerr/discussions/341#discussioncomment-5213305>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOWBYVS2BVNIUV56D653RKDW2WSU3ANCNFSM6AAAAAAVQTWMZ4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
I'll be more willing to encourage thinking outside the box when I see some
evidence of thinking inside the box.
-Terry Pratchett

**Reply by fallenbagel** [reply]
Sounds like LDAP is the issue. We have OIDC PR ongoing rn but as of currently LDAP/SSO/OIDC is not yet supported in jellyseerr

**Comment by fliberdygibits** [comment]
It's been some time since i set this up.  Is my main account... my personal account also the "admin" account that's being used by jellyseerr to talk to jellyfin?  Could I just create a different non-ldap jellyfin user to use as the admin?
